playwright.config.ts

```diff
@@ -4,7 +4,7 @@ import { autoTestConfig } from './appSetting.json';
 
 const config: PlaywrightTestConfig = {
   
-  retries: 2,
+  retries: 3,
```

Comment:
Please check if these changes to playwright config should be included

Reply:
Reverted the changes, Thanks Chris.